### PR TITLE
Tech Writer updates: spacing/indentation, escaped special characters,…

### PR DESCRIPTION
**Pull request by DannaShirley**
… accessibility text, etc.

### CODE REVIEW

File: blueprint/index.md [hunk 1]
```diff
@@ -1,171 +1,176 @@
...
+3. (Optional) Use the **Name** box to give the widget a meaningful name. For example, **Chat Translator**.
+
+   ![Interaction Widget name](images/name-interaction.png "Interaction Widget name")
+
+4. Click the **Configuration** tab.
+5. In the **Application URL** box, type the URL of the web application. Be sure to specify the full URL, beginning with `https:`.
+
+  ```
+   https://localhost/?conversationid=&#123;&#123;pcConversationId&#125&#125;&language=&#123;&#123;pcLangTag&#125&#125;
+  ```
```
Comment (AgnesCorpuz) [inline blueprint/index.md:102]: The new Developer Center can now properly render the  double curly `{{ }}`. We may now have this replaced to `https://localhost/?conversationid={{pcConversationId}}&language={{pcLangTag}}`

Indented code fences also do not render properly in the Developer Center. We need to remove the spaces here for it to render properly.

File: blueprint/index.md [hunk 2]
```diff
@@ -1,171 +1,176 @@
...
-
-Clicking the **Open Canned Responses** above the message box allows an agent to send canned response translated to the customer's language.
-![Translate Canned Response](images/translate-canned-response.png "Translate Canned Response")
-
-## Additional Resources
-
-- [Genesys Cloud Developer Center](https://developer.mypurecloud.com/)
-- [Genesys Cloud Platform Client SDK](https://developer.mypurecloud.com/api/rest/client-libraries/)
\ No newline at end of file
+  ```
```
Comment (AgnesCorpuz) [inline blueprint/index.md:135]: Indented code fence render issue also applies here

File: blueprint/index.md [hunk 3]
```diff
@@ -1,171 +1,176 @@
...
+2. Switch to the directory where the files for your Chat Translator project are located and install the dependencies in the local node-modules folder. In the command line, type `npm install`.
+3. To run the server locally, in the command line type `node run-local.js`.
+
+### Test the solution
+
+1. Create a Genesys web chat widget. For more information, see [Create a widget for web chat](https://help.mypurecloud.com/?p=195772 "Opens the Create a widget for web chat article") in the Genesys Cloud Resource Center.
+
+  :::primary
+  **Important**: If you will use the Genesys Cloud developer tools to test this solution, then make sure that under **Widget Type** you select **Version 2**, **Version 1.1**, or **Third Party**. For more information see [About widgets for web chat](https://help.mypurecloud.com/articles/?p=194115 "Opens the About widgets for web chat article") in the Genesys Cloud Resource Center.
+  :::
```
Comment (AgnesCorpuz) [inline blueprint/index.md:152]: Indented alert block render issue also applies here

File: blueprint/index.md [hunk 4]
```diff
@@ -1,171 +1,176 @@
...
+
+2. Open the [Web Chat developer tool](https://developer.mypurecloud.com/developer-tools/#/webchat "Opens the Web Chat developer tool").
+
+  :::primary
+  **Important**: Make sure the Developer Center URL matches the region where your Genesys Cloud organization is located. For more information, see the [Access the developer tools](https://developer.mypurecloud.com/gettingstarted/developer-tools-intro.html#accessTools "Goes to the Access the developer tools section on the Developer tools quick start page") section on the Developer tools quick start page.
+  :::
```
Comment (AgnesCorpuz) [inline blueprint/index.md:158]: Indented alert block render issue also applies here

**Review by AgnesCorpuz — COMMENTED** [review]
Hey Danna, Developer Center won't render indented alert box and code fences properly. You may want to test how Developer Center will render the markdown file [here](https://apicentral.inindca.com/services/developer-center/editor).

File: blueprint/index.md [hunk 5]
```diff
@@ -1,171 +1,176 @@
...
 
-3. **(Optional)** Use the Name box to give the app a different name.
-   ![Name Interaction Widget](images/name-interaction.png "Name Interaction Widget")
-   
-4. Click the **Configuration** tab. In the **Application URL** box, type the URL of the web application. Be sure to specify the full URL, including the https: at the beginning of the URL.
-   
-   The pcConversationId URL parameter is used to determine the conversation interaction and pcLangTag is used to determine the agent's language in the Chat Translator solution.
+  :::primary
+  **Tip**: Start with the sample.env file for this blueprint and provide your org-specific details.
+  :::
```
[the remark this answers is not in the document]
Comment (DannaShirley) [reply blueprint/index.md:81]: Thanks Agnes. It's my understanding that Tim has a PR open to support indented alerts. @timsmithgenesys Can you please confirm?

[the remark this answers is not in the document]
Comment (timsmithgenesys) [reply blueprint/index.md:81]: Yes, indentation is supported. The docs for it aren't published yet, but I'll send you a link to the doc source in bitbucket. (please don't link to internal resources like bitbucket from public sites like this)

[the remark this answers is not in the document]
Comment (DannaShirley) [reply blueprint/index.md:81]: Sorry about that!

**Pull request merged**